app/views/users/new.html.erb

```diff
@@ -0,0 +1,58 @@
+<% if @user.errors.any? %>
+  <div id="error_explanation">
+    <h2><%= pluralize(@user.errors.count, "error") %> prohibited this user from being saved:</h2>
+    <ul>
+      <% @user.errors.each do |error| %>
+        <li><%= error.full_message %></li>
+      <% end %>
+    </ul>
+  </div>
+<% end %>
+<%= form_with model: @user do |form| %>
+  <%= label_tag(:username, "Username") %>
+  <%= form.text_field :username, placeholder: "Your user name here" %>
+  <br>
+  <br>
+  <%= label_tag(:email, "Email") %>
+  <%= form.text_field :email, placeholder: "example@example.com" %>
+  <br>
+  <br>
+  <%= label_tag(:password, "Password") %>
+  <%= form.password_field :password %>
+  <br>
+  <br>
+  <%= form.submit "Create" %>
+<% end %>
+<!--<hr>
+<form method="post" action="/users" accept-charset="UTF-8" data-remote="true">
```

Comment:
[required] As described in the first review, the action should be `action="users_path"` not ~action="/users"~